packages/contracts/contracts/gatekeepers/SignUpGatekeeper.sol

```diff
@@ -1,20 +1,16 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.28;
 
+import { Policy } from "@excubiae/contracts/policy/Policy.sol";
+import { IBasePolicy } from "@excubiae/contracts/interfaces/IBasePolicy.sol";
+
 /// @title SignUpGatekeeper
 /// @notice A gatekeeper contract which allows users to sign up for a poll.
-abstract contract SignUpGatekeeper {
+abstract contract SignUpGatekeeper is Policy, IBasePolicy {
```

Comment:
why are we doing this here? let's convert gatekeepers and push them to the excubiae repo as extensions, so that others can reuse them too

Reply:
yea we should be making those checkers in the excubiae repo (as extensions), so that we can import them here (and others can reuse them too)